utils/docker/dev/Dockerfile

```diff
@@ -86,9 +86,13 @@ RUN git clone --branch v$FABRIC_VERSION $FABRIC_REPO $FABRIC_PATH
 ENV FPC_PATH=$GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode
 # We copy context so we can use that to potentially get local .git as repo ...
 COPY .git /tmp/cloned-local-fpc-git-repo
-RUN git clone --recurse $FPC_REPO_URL $FPC_PATH \
+RUN git \
+       -c submodule.interpreters/wasm-micro-runtime.update=none -c submodule.ccf_transaction_processor/CCF.update=none \
+	clone --recurse $FPC_REPO_URL $FPC_PATH \
   && cd $FPC_PATH \
   && git checkout --recurse $FPC_REPO_BRANCH_TAG_OR_COMMIT
+# Notes:
+# - the -c submodule's are to prevent dragging in large but unneeded sub-sub-modules of pdo ...
```

Comment:
same, comment before doing things.

Reply:
Right now in dockerfiles (and also in scripts afaik) such "Notes:" comments are always after the command.  So for consistency reason i would leave it (unless we think it's worth the effort to consistently change it to opposite order)?

> same, comment before doing things.

Right now in dockerfiles (and also in scripts afaik) such "Notes:" comments are always after the command.  So for consistency reason i would leave it (unless we think it's worth the effort to consistently change it to opposite order)?